youslist.txt

```diff
@@ -1075,6 +1075,7 @@
 ||busan.com/nas/rssimage/adrss6.html^$domain=m.busan.com
 ||busan.com/w2010/AD/$domain=busan.com|mobile.busan.com
 ||busygirlblog.com/adserver/$domain=busygirlblog.com
+||buzz-js.buzzvil.com/buzzvil-pixel-sdk/*.js^
```

Comment:
I think `||buzz-js.buzzvil.com/buzzvil-pixel-sdk/buzzvil-pixel.js^` will be okay.

Reply:
I accepted your suggestion! Thanks. Waiting for CI to check.